app/src/main/java/org/simple/clinic/teleconsultlog/medicinefrequency/MedicineFrequency.kt

```diff
@@ -1,8 +1,53 @@
 package org.simple.clinic.teleconsultlog.medicinefrequency
 
-enum class MedicineFrequency {
-  OD,
-  BD,
-  TDS,
-  QDS
+import android.os.Parcelable
+import androidx.room.TypeConverter
+import com.squareup.moshi.FromJson
+import com.squareup.moshi.ToJson
+import kotlinx.android.parcel.Parcelize
+import org.simple.clinic.util.room.SafeEnumTypeAdapter
+
+sealed class MedicineFrequency : Parcelable {
```

Comment:
Since we change to a sealed class from an enum, the earlier `.toString()` implementations will be broken since this will be different for every instance of the JVM process. Can we manually override the `.toString` for each of the `object` types to make them constant?

Reply:
we weren't using `.toString()` for these classes, should I still override that method? 🤔

Reply:
They will be used implicitly if we print any data classes which contain these properties as part of a string template.